modules/ROOT/pages/index.adoc

```diff
@@ -187,9 +256,34 @@ include::partial$common.adoc[tag="windows-separators-warning"]
 
 --
 
+Capella Columnar::
++
+--
+. Open the ODBC Administration tool.
+. Create a new ODBC connection using one of the Couchbase drivers (either `ANSI` or `Unicode`), then select btn:[Finish]
+. On the next window, select btn:[Capella Columnar].
+. Enter the following details in the ODBC configuration dialog:
++
+[horizontal]
+*Name:*:: Enter an identifying name for the data source.
+*Description:*:: Enter an optional description.
+*URL:*:: Enter the connection string *exactly*  as copied <<capella-columnar-connection-string,here>>.
```

Comment:
It's actually aiming for the top of the section where copying the connection string begins. I think it's better to link the whole section, rather than just the instruction which tells them to copy the string.